app/jobs/appointment_notification/worker.rb

```diff
@@ -38,14 +38,15 @@ def perform(notification_id)
   private
 
   def send_message(notification, communication_type)
-    notification_service = if notification.experiment&.experiment_type == "medication_reminder" && medication_reminder_sms_sender
+    notification_service = if communication_type == "imo"
+      ImoApiService.new
+    # given that we expect higher volume in the future, should this be default behavior?
+    elsif notification.experiment&.experiment_type == "medication_reminder" && medication_reminder_sms_sender
```

Comment:
given that we expect higher volume in the future, should this be default behavior?